src/Command/FilesCommand.php

```diff
@@ -0,0 +1,152 @@
+            )
+            ->addArgument(
+                FindAndUploadFilesCommand::ARGUMENT_PASSWORD,
+                InputArgument::REQUIRED,
+                'Your Debricked password or access token',
+            )
+            ->addArgument(
+                FindAndUploadFilesCommand::ARGUMENT_BASE_DIRECTORY,
+                InputArgument::REQUIRED,
+                'The base directory (relative to current working directory) to recursively find dependency files in. Default is current working directory.',
+            )
+            ->addOption(
+                'json',
+                null,
+                InputOption::VALUE_NONE,
+                'Use this option to output files in json'
+            )
+            ->addOption(
+                FindAndUploadFilesCommand::OPTION_RECURSIVE_FILE_SEARCH,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Set to 0 to disable recursive search - only base directory will be searched',
+                1
+            )
+            ->addOption(
+                FindAndUploadFilesCommand::OPTION_DIRECTORIES_TO_EXCLUDE,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Enter a comma separated list of directories to exclude. Such as: --excluded-directories="vendor,node_modules,tests"',
+                'vendor,node_modules,tests'
+            );
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $api = new API(
+            $this->debrickedClient,
+            \strval($input->getArgument(FindAndUploadFilesCommand::ARGUMENT_USERNAME)),
+            \strval($input->getArgument(FindAndUploadFilesCommand::ARGUMENT_PASSWORD))
+        );
+
+        $baseDirectory = $input->getArgument(FindAndUploadFilesCommand::ARGUMENT_BASE_DIRECTORY);
+        $workingDirectory = \getcwd();
+        if ($workingDirectory === false) {
+            $io->warning(
+                'Failed to get current working directory, command might be searching through unexpected directories and files'
+            );
+        }
+        $searchDirectory = "{$workingDirectory}/{$baseDirectory}";
+        $searchDirectory = preg_replace('#/+#', '/', $searchDirectory); // remove duplicate slashes.
+
+        $json = (bool) $input->getOption(FilesCommand::OPTION_JSON);
+        $recursiveFileSearch = (bool) $input->getOption(FindAndUploadFilesCommand::OPTION_RECURSIVE_FILE_SEARCH);
+        $directoriesToExcludeString = $input->getOption(FindAndUploadFilesCommand::OPTION_DIRECTORIES_TO_EXCLUDE);
+        $directoriesToExcludeArray = [];
+        if (empty($directoriesToExcludeString) === false) {
+            $directoriesToExcludeArray = \explode(',', $directoriesToExcludeString) ?? [];
+        } else {
+            $io->note('No directories will be ignored');
+        }
```

Comment:
Maybe to create a private method with a descriptive name instead of putting full logic here?